Documentation/cmdref/cilium-agent.md

```diff
@@ -63,6 +63,7 @@ cilium-agent [flags]
       --enable-ipv6                                           Enable IPv6 support (default true)
       --enable-k8s-event-handover                             Enable k8s event handover to kvstore for improved scalability
       --enable-node-port                                      Enable NodePort type services by Cilium (beta)
+      --enable-non-local-node-identity                        Enable use of non-local node identity
```

Comment:
Can you add in the commit message why is it important to maintain backwards compatibility? In other words, why is it important that we won't start using this identity right away by default?

Reply:
Done